footprints/main/models.py

```diff
@@ -994,8 +969,15 @@ def display_title(self):
             return self.title
 
     def owners(self):
-        role = Role.objects.get_owner_role()
-        return self.actor.filter(role=role)
+        return self.actor.filter(
+            role__name=Role.OWNER).select_related('person')
+
+    def owners_list(self):
+        a = ''
+        for actor in self.actor.all():
+            if actor.role.name == Role.OWNER:
+                a += actor.display_name() + ', '
+        return a[:-2]
```

Comment:
Right, I know this looks awkward and is inefficient from a string concatenation viewpoint. Unfortunately, completing the self.actor.filter here defeats [this prefetch_related](https://github.com/ccnmtl/footprints/pull/356/files#diff-c96bb4a9eaac6708222cf9139b504eefR149,) turning the request from 100ms/40 queries into 16000ms + thousands of queries. Other approaches/thoughts welcome.